answers.go

```diff
@@ -33,19 +33,23 @@ type AnswerResponse struct {
 }
 
 // Search — perform a semantic search api call over a list of documents.
-func (c *Client) Answers(ctx context.Context, request AnswerRequest) (response AnswerResponse, err error) {
+func (c *Client) Answers(ctx context.Context, request AnswerRequest) (AnswerResponse, error) {
```

Comment:
I'm not sure this helps code readability, maybe we should just relax linter settings?

Curious why Github's golangci-lint did not report this.

Reply:
I added a `.golangci.yaml` file which has a bunch of settings enabled, one of which is no-named imports. We can just disable it